release/gcb/generate_manifest.sh

```diff
@@ -53,19 +53,23 @@ function create_manifest_check_consistency() {
  pushd istio
   local ISTIO_REPO_SHA
   local PROXY_REPO_SHA
+  local CNI_REPO_SHA
   local API_REPO_SHA
   ISTIO_REPO_SHA=$(git rev-parse HEAD)
+  #TODO add proper tracking of the SHA
+  CNI_REPO_SHA=3b977f8c6fe9c694b99617be0ae77dec11f50dde
```

Comment:
@rkpagadala - thanks for your review.  Yes it does absolutely change with time that is why I added the TODO,  but we need a general strategy on how we want to version the CNI relative to Istio.  See email from Tim Swanson.   I wanted to put something fixed here for now so that we could at least iterate on things.